docs/guides/publish.md

```diff
@@ -62,6 +62,19 @@ parallel uploads. Note that existing files need to match exactly with those prev
 the registry, this avoids accidentally publishing source distribution and wheels with different
 contents for the same version.
 
+## Private Packages
```

Comment:
This doesn't quite make sense in the publishing guide, which is explicitly about how to publish your package. I guess I'll move towards having a publish concept document, but in the meantime, it can live here. It's a little awkward in the middle of this workflow, maybe it should be in `Preparing your project for packaging` as a `!!! tip`?

Could you add a sentence at the top of this section which explains what a private package is and why you'd want one?

Reply:
i moved it up again and changed word to "internal", that's self-explanatory.